ng2-components/ng2-activiti-form/README.md

```diff
@@ -315,6 +315,7 @@ class MyComponent {
 | taskSaved | FormEvent | Raised when a task is saved successfully |
 | taskSavedError | FormErrorEvent | Raised when a task is saved unsuccessfully |
 | executeOutcome | FormOutcomeEvent | Raised when a form outcome is executed |
+| formEvents | Event | You can subscribe to this event to listen : ( click, blur, change, focus, focusin, focusout, input, invalid, select) of any elements in the form , see doc below|
```

Comment:
it retrurns the true event get from the dom and redirect therefore the type is Event